File: unidbg-api/src/main/java/com/vortexdbg/debugger/gdb/KillCommand.java
package com.vortexdbg.debugger.gdb;

import com.vortexdbg.Emulator;

class KillCommand implements GdbStubCommand {

    @Override
    public boolean processCommand(Emulator<?> emulator, GdbStub stub, String command) {
        stub.send("+");
        stub.shutdownServer();
        System.exit(9);
        return true;
    }

}

File: vortexdbg-api/src/main/kotlin/com/vortexdbg/debugger/gdb/KillCommand.kt
package com.vortexdbg.debugger.gdb

import com.vortexdbg.Emulator

internal class KillCommand : GdbStubCommand {

    override fun processCommand(emulator: Emulator<*>, stub: GdbStub, command: String): Boolean {
        stub.send("+")
        stub.shutdownServer()
        System.exit(9)
        return true
    }

}
